src/NLog/LayoutRenderers/StackTraceFormat.cs

```diff
@@ -52,5 +52,10 @@ public enum StackTraceFormat
         /// Detailed flat format (method signatures displayed in a single line).
         /// </summary>
         DetailedFlat,
+
+        /// <summary>
+        /// Raw format (multiline with source info - as returned by StackFrame.ToString() method).
+        /// </summary>
+        RawWithSource,
```

Comment:
I'm doubting if we should patch `raw` instead of a new option (I know i proposed it earlier ;))

Printing ` file:line:column <filename unknown>:0:0  ` sounds useless to me and the default is `flat`. 

What do you think?